.circleci/config.yml

```diff
@@ -4,28 +4,14 @@ jobs:
     docker:
       - image: fpco/stack-build:lts
     steps:
-      - checkout
-      - restore_cache:
-          keys:
-            - stack-ghc-{{ checksum "stack.yaml" }}
-      - restore_cache:
-          keys:
-            - stack-deps-{{ checksum "package.yaml" }}
-      - run:
-          name: Set up Stack
-          command: stack setup --no-terminal --no-reinstall
-      - save_cache:
-          key: stack-ghc-{{ checksum "stack.yaml" }}
-          paths:
-            - /root/.stack
       - run:
-          name: Install dependencies
-          command: stack build --skip-ghc-check --no-terminal --test --only-dependencies
-      - save_cache:
-          key: stack-deps-{{ checksum "package.yaml" }}
-          paths:
-            - /root/.stack
-            - .stack-work
+          name: Install Dependencies
+          command: |
+            apt-get update
```

Comment:
pretty minor but I think you're "supposed" to spell this `apt update` and `apt install` these days (I have no idea when this way might stop working, if ever though).

Reply:
I'd like to replace this CI config with something that doesn't need to run `apt update` in each job so I won't worry about this too much for now, I think.

Also, for a while, wasn't `apt` the "unstable, user-facing interface" while `apt-get` was still the "stable, scriptable interface"?  But it has been a few years since I did much on Debian so I'm not sure if I remember correctly, or if it is still this way.